DetDescr/src/DetDescr/EcalGeometry.cxx

```diff
@@ -1,16 +1,5 @@
 #include "DetDescr/EcalGeometry.h"
 
-#include <assert.h>
-
-#include <iomanip>
-#include <iostream>
-
-#include "TGeoPolygon.h"
-#include "TGraph.h"
-#include "TList.h"
-#include "TMath.h"
-#include "TMultiGraph.h"
-
```

Comment:
Why are you moving the includes here into the header? Generally, more includes leads to longer build times (any file that includese EcalGeometry.h will then need to also parse TGeoPolygon, TGraph,...) and so if we can isolate includes to the source files that require them, we should.

Reply:
Hmm I see your point, but like this we actually had duplicates. So keeping everything in the header avoids that.

Reply:
Fair enough, I'll leave this in favor of using `clang-tidy` or `iwyu` to algorithmically do it for us in the future.